src/components/CallView/CallView.vue

```diff
@@ -151,12 +152,16 @@ import ViewerOverlayCallView from './shared/ViewerOverlayCallView.vue'
 
 import { placeholderImage, placeholderModel, placeholderName, placeholderSharedData } from './Grid/gridPlaceholders.ts'
 import { SIMULCAST } from '../../constants.js'
+import BrowserStorage from '../../services/BrowserStorage.js'
 import { fetchPeers } from '../../services/callsService.js'
 import { getTalkConfig } from '../../services/CapabilitiesManager.ts'
 import { EventBus } from '../../services/EventBus.js'
 import { localMediaModel, localCallParticipantModel, callParticipantCollection } from '../../utils/webrtc/index.js'
 import RemoteVideoBlocker from '../../utils/webrtc/RemoteVideoBlocker.js'
 
+const serverVersion = loadState('core', 'config', {}).versionstring ?? '29.0.0'
```

Comment:
![image](https://github.com/user-attachments/assets/1da50976-420c-46ff-a78f-0d184fec3a78)
`version` seems to be a better option